src/components/nav.jsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+
+class Nav extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+  render() {
+    return (
+      <div>
+        <ul className='nav justify-content-end'>
+          <li
+            className={
+              this.props.cssBadge === 'view-cards'
```

Comment:
Everywhere you are using `cssBadge` you can just use `view` since their values are always the same.